### ISSUE TITLE
Could not find package altis/skeleton with stability stable.

### ISSUE THREAD

**Issue by BronsonQuick** [humanmade/altis-skeleton#12]
I was just going through the developer documentation to setup a new Altis project locally and after running `composer create-project altis/skeleton altis` I received the following error:
```
[InvalidArgumentException]                                    
  Could not find package altis/skeleton with stability stable.
```

<img width="1429" alt="Sites__fish_VolumesSites__-fish__15841_2019-05-22_10-56-27" src="https://user-images.githubusercontent.com/1377956/58140282-f2582880-7c81-11e9-87c2-d0abdf9587fb.png">

**Comment by BronsonQuick** [comment]
Here's the output of `composer create-project altis/skeleton altis -vvv` in case you need it.

```
Reading /Users/bronson/.composer/composer.json
Loading config file /Users/bronson/.composer/composer.json
Checked CA file /private/etc/ssl/cert.pem: valid
Executing command (/Users/bronson/.composer): git branch --no-color --no-abbrev -v
Executing command (/Users/bronson/.composer): git describe --exact-match --tags
Executing command (/Users/bronson/.composer): git log --pretty="%H" -n1 HEAD
Executing command (/Users/bronson/.composer): hg branch
Executing command (/Users/bronson/.composer): fossil branch list
Executing command (/Users/bronson/.composer): fossil tag list
Executing command (/Users/bronson/.composer): svn info --xml
Reading /Users/bronson/.composer/vendor/composer/installed.json
Running 1.8.5 (2019-04-09 17:46:47) with PHP 7.3.4 on Darwin / 18.5.0
Reading /Users/bronson/.composer/composer.json
Loading config file /Users/bronson/.composer/composer.json
Executing command (/Users/bronson/.composer): git branch --no-color --no-abbrev -v
Executing command (/Users/bronson/.composer): git describe --exact-match --tags
Executing command (/Users/bronson/.composer): git log --pretty="%H" -n1 HEAD
Executing command (/Users/bronson/.composer): hg branch
Executing command (/Users/bronson/.composer): fossil branch list
Executing command (/Users/bronson/.composer): fossil tag list
Executing command (/Users/bronson/.composer): svn info --xml
Reading /Users/bronson/.composer/vendor/composer/installed.json
Downloading https://repo.packagist.org/packages.json
Writing /Users/bronson/.composer/cache/repo/https---repo.packagist.org/packages.json into cache
Reading /Users/bronson/.composer/cache/repo/https---repo.packagist.org/p-provider-2013.json from cache
Reading /Users/bronson/.composer/cache/repo/https---repo.packagist.org/p-provider-2014.json from cache
Reading /Users/bronson/.composer/cache/repo/https---repo.packagist.org/p-provider-2015.json from cache
Reading /Users/bronson/.composer/cache/repo/https---repo.packagist.org/p-provider-2016.json from cache
Downloading http://repo.packagist.org/p/provider-2017%24bf028610a76cb0c58a45ee7edef651f2f3f16d32cd1f7a93000f519336fa011a.json
Writing /Users/bronson/.composer/cache/repo/https---repo.packagist.org/p-provider-2017.json into cache
Reading /Users/bronson/.composer/cache/repo/https---repo.packagist.org/p-provider-2018.json from cache
Reading /Users/bronson/.composer/cache/repo/https---repo.packagist.org/p-provider-2018-07.json from cache
Reading /Users/bronson/.composer/cache/repo/https---repo.packagist.org/p-provider-2018-10.json from cache
Reading /Users/bronson/.composer/cache/repo/https---repo.packagist.org/p-provider-2019-01.json from cache
Downloading http://repo.packagist.org/p/provider-2019-04%247f3c9030ed6f49c7dda267052dadad6d163d479613c5554501c75245fdbeb5fe.json
Writing /Users/bronson/.composer/cache/repo/https---repo.packagist.org/p-provider-2019-04.json into cache
Reading /Users/bronson/.composer/cache/repo/https---repo.packagist.org/p-provider-archived.json from cache
Downloading http://repo.packagist.org/p/provider-latest%247412fbe37476b3523f72bf51131e01fe631c328c8e13604165594733304d3a88.json
Writing /Users/bronson/.composer/cache/repo/https---repo.packagist.org/p-provider-latest.json into cache
Reading /Users/bronson/.composer/cache/repo/https---repo.packagist.org/provider-altis$skeleton.json from cache

                                                                
  [InvalidArgumentException]                                    
  Could not find package altis/skeleton with stability stable.  
                                                                

Exception trace:
 () at phar:///usr/local/Cellar/composer/1.8.5/bin/composer/src/Composer/Command/CreateProjectCommand.php:318
 Composer\Command\CreateProjectCommand->installRootPackage() at phar:///usr/local/Cellar/composer/1.8.5/bin/composer/src/Composer/Command/CreateProjectCommand.php:159
 Composer\Command\CreateProjectCommand->installProject() at phar:///usr/local/Cellar/composer/1.8.5/bin/composer/src/Composer/Command/CreateProjectCommand.php:145
 Composer\Command\CreateProjectCommand->execute() at phar:///usr/local/Cellar/composer/1.8.5/bin/composer/vendor/symfony/console/Command/Command.php:245
 Symfony\Component\Console\Command\Command->run() at phar:///usr/local/Cellar/composer/1.8.5/bin/composer/vendor/symfony/console/Application.php:835
 Symfony\Component\Console\Application->doRunCommand() at phar:///usr/local/Cellar/composer/1.8.5/bin/composer/vendor/symfony/console/Application.php:185
 Symfony\Component\Console\Application->doRun() at phar:///usr/local/Cellar/composer/1.8.5/bin/composer/src/Composer/Console/Application.php:258
 Composer\Console\Application->doRun() at phar:///usr/local/Cellar/composer/1.8.5/bin/composer/vendor/symfony/console/Application.php:117
 Symfony\Component\Console\Application->run() at phar:///usr/local/Cellar/composer/1.8.5/bin/composer/src/Composer/Console/Application.php:104
 Composer\Console\Application->run() at phar:///usr/local/Cellar/composer/1.8.5/bin/composer/bin/composer:61
 require() at /usr/local/Cellar/composer/1.8.5/bin/composer:24

create-project [-s|--stability STABILITY] [--prefer-source] [--prefer-dist] [--repository REPOSITORY] [--repository-url REPOSITORY-URL] [--dev] [--no-dev] [--no-custom-installers] [--no-scripts] [--no-progress] [--no-secure-http] [--keep-vcs] [--remove-vcs] [--no-install] [--ignore-platform-reqs] [--] [<package>] [<directory>] [<version>]

```

**Comment by BronsonQuick** [comment]
For now I'm just working around it with `composer create-project altis/skeleton altis 1.0.0-rc2`

**Comment by rmccue** [comment]
This is just as we haven't published 1.0.0 final yet, so `composer create-project altis/skeleton:1.0.0-rc2`
